pylti1p3/lineitem.py

```diff
@@ -221,16 +233,25 @@ def set_submission_type(self, _type: str, external_tool_url: t.Optional[str]) ->
         return self
 
     def get_value(self) -> str:
+        """
+        Prepare the line item data in the format required by the LTI-AGS specification.
+
+        The final dictionary is then converted to a JSON string and then sent via the API.
+        Any missing values are explicitly set to `None` in the dictionary and then converted to
+        `null` values in the JSON.
+        https://www.imsglobal.org/spec/lti-ags/v2p0/#updating-a-line-item
+        https://www.imsglobal.org/spec/lti-ags/v2p0/openapi/#/default
+        """
         data = {
-            "id": self._id if self._id else None,
```

Comment:
So it relies only on the `resourceId` to match and do the `PUT` operation?

Reply:
The `id` is set in the URL we're issuing `PUT` request to. The LTI standard specifies that you can put it in the payload, but there's zero reason for that.